src/config/config.js

```diff
@@ -352,6 +353,16 @@ const production = {
                     isQuestioning: '1109912874459078867',
                     isCis: '1109912938929737909',
                 },
+            links:  {
+                rules: 'https://canary.discord.com/channels/959551566388547676/1057132419150532678/1151892231091925163',
```

Comment:
this links to TP right now because there's no way to link the message before it's sent. TSO theo can't currently send the message with the configuration missing, so this will be edited after the message is sent in a separate PR